vuepress/docs/next/tutorials/create/mfe/angular.md

```diff
@@ -74,245 +88,172 @@ This is the expected output:
     │   ├── styles.css
     │   └── test.ts
     │
+    ├── .browserlistrc
     ├── .editorconfig
     ├── .gitignore
     ├── angular.json
-    ├── browserlist
     ├── karma.conf.js
     ├── package.json
+    ├── package-lock.json
     ├── README.md
     ├── tsconfig.app.json
     ├── tsconfig.json
-    ├── tsconfig.spec.json
-    └── tslint.json
+    └── tsconfig.spec.json
+    
 
-### Convert to Custom Element
-
-Next, let's convert our Angular app into a custom element. We'll use [Angular elements](https://angular.io/guide/elements) to transform components into custom elements.
+### Configure the Custom Element
 
+Next, convert the Angular widget into a custom element, using [Angular elements](https://angular.io/guide/elements) to transform components into custom elements.
+1. Install the Angular elements
 ``` bash
 ng add @angular/elements
 ```
+[Angular elements](https://angular.io/guide/elements) are Angular components packaged as custom elements (also called Web Components), a web standard for defining new HTML elements in a framework-agnostic way.
 
 ::: warning
-Install the Angular elements package using `ng add`, not with `npm install` as it runs additional steps behind the scenes like adding the `document-register-element` polyfill.
+Install the Angular elements package using `ng add`, not with `npm install` as it runs additional steps behind the scenes.
```

Comment:
it might be better to put the "not with `npm install`..." bit in parentheses; this part is not essential to the instruction and draws attention away from what to do. the focus should be on the first part of the sentence, even tho the note is useful.